src/material/dialog/dialog-container.ts

```diff
@@ -281,7 +281,7 @@ export class MatDialogContainer extends _MatDialogContainerBase {
   _onAnimationStart({toState, totalTime}: AnimationEvent) {
```

Comment:
@crisbeto I saw that above in `_onAnimationDone` we potentially could also handle `void`. That hasn't been the case before either so I'm leaving it as is (seems like something worth exploring in the future)

Reply:
This might be more appropriate in `_animationDone`. I haven't tried running it, but it seems like doing it at the start animation might cut off the backdrop transition.

Reply:
I might be a bit confused. Are you saying we should only handle `void` in the animation done handler? With the comment above I actually just meant to say that we could also handle the `void` state above `end` (while keeping the one in start too). The way it is right now matches the behavior of the `@angular/material/dialog` exactly as before my refactor. The backdrop will be detached when the `closing` state is emitted (as before my refactor).

Reply:
ok, I see that we have `closing` and `closed` states. When I read it the first time around it seemed like this would emit that the closing sequence is finished.

Reply:
Got it. Thanks for checking 👍